enlighten/measurement/Measurement.py

```diff
@@ -684,20 +684,30 @@ def rename_files(self):
 
     ## @todo cloud etc
     def save(self):
+        saved = False
         if self.save_options.save_csv():
             self.save_csv_file()
+            saved = True
 
         if self.save_options.save_text():
             self.save_txt_file()
+            saved = True
 
         if self.save_options.save_excel():
             self.save_excel_file()
+            saved = True
 
         if self.save_options.save_json():
             self.save_json_file()
+            saved = True
 
         if self.save_options.save_spc():
             self.save_spc_file()
+            saved = True
+
+        if not saved:
+            if self.measurements:
+                self.measurements.marquee.error("No save formats selected -- spectrum not saved to disk")
```

Comment:
This was leading users to believe they were saving data when they weren't

Reply:
if, elif, else